dev-cpp/abseil-cpp/files/abseil-cpp-20240116.2-include-cstdint.patch

```diff
@@ -0,0 +1,33 @@
+https://github.com/abseil/abseil-cpp/commit/809e5de7b92950849289236a5a09e9cb4f32c7b9
```

Comment:
As reviewed on IRC but for the benefit of others: needs a revbump as it affects installed headers in this case.